### ISSUE TITLE
FSM blueprint needs to be updated with client name that does not contain spaces

### ISSUE THREAD

**Issue by JakeSchmitz** [Vantiq/Blueprints#12]
It is no longer legal to have a client name contain spaces. The Service Desk client violates this restriction and should be updated to have a name like `Service-Desk` instead.

@pburma do you want to make this change? I can make it if you do not have time, but is there anything else that needs to be adjusted besides the name? Are there references to the client by name elsewhere in the blueprint?

**Comment by AdrianoGomesRJ** [comment]
FYI... it's still an issue... :-) I just tried to upload it, and I had to manualy fix the "Service Desk" page set file name and the "name" property inside the file.

**Comment by realtdegen** [comment]
I've updated "Service Desk" client to "ServiceDesk" and submitted a pull request into master, now I wonder who is going to review and approve this one to get the updated fileset out to the public

**Comment by realtdegen** [comment]
well, I've tried to commit into Vantiq Blueprints Git on master but need "At least 1 approving review is required by reviewers with write access.".....so question is : Who is an authorized reviewer with write access there ?

Enumerating objects: 21, done.
Counting objects: 100% (21/21), done.
Delta compression using up to 8 threads
Compressing objects: 100% (15/15), done.
Writing objects: 100% (15/15), 548.34 KiB | 7.31 MiB/s, done.
Total 15 (delta 9), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (9/9), completed with 5 local objects.        
remote: error: GH006: Protected branch update failed for refs/heads/master.        
remote: error: At least 1 approving review is required by reviewers with write access.        
To https://github.com/Vantiq/Blueprints.git
 ! [remote rejected] master -> master (protected branch hook declined)
error: failed to push some refs to 'https://github.com/Vantiq/Blueprints.git'

**Comment by rblumer4** [comment]
I don't see any pull request for this change. 
How did you create your PR?

To create a PR we can see in github, "git push" your branch and then use the github web UI to create a PR from that branch to Vantiq/Blueprints:master.  From either the Code page or the Pull Requests page.

**Comment by realtdegen** [comment]
you are right, I ~thought~ having created a pull request but only tried to commit to master which got refused

Enumerating objects: 21, done.
Counting objects: 100% (21/21), done.
Delta compression using up to 8 threads
Compressing objects: 100% (15/15), done.
Writing objects: 100% (15/15), 548.34 KiB | 18.28 MiB/s, done.
Total 15 (delta 9), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (9/9), completed with 5 local objects.        
remote: error: GH006: Protected branch update failed for refs/heads/master.        
remote: error: At least 1 approving review is required by reviewers with write access.        
To https://github.com/Vantiq/Blueprints.git
 ! [remote rejected] master -> master (protected branch hook declined)
error: failed to push some refs to 'https://github.com/Vantiq/Blueprints.git'

**Comment by realtdegen** [comment]
https://github.com/Vantiq/Blueprints/pull/21 created for review and merge
